myfitnesspal/exercise.py

```diff
@@ -22,6 +22,14 @@ def entries(self):
     def name(self):
         return self._name
 
+    @property
+    def totals(self):
+        exercises = {'minutes': 0, 'calories burned': 0}
```

Comment:
Could I convince you into adding support for `unit_aware` values for these fields?  If that doesn't mean anything to you, you might be able to glean a little bit of information [from the main client class](https://github.com/coddingtonbear/python-myfitnesspal/blob/master/myfitnesspal/client.py).

Reply:
I tried it with unit_aware first and got values that did not make sense to me. I will have another look into it.